expression/builtin_arithmetic_vec.go

```diff
@@ -551,9 +551,40 @@ func (b *builtinArithmeticPlusDecimalSig) vecEvalDecimal(input *chunk.Chunk, res
 }
 
 func (b *builtinArithmeticMultiplyIntUnsignedSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinArithmeticMultiplyIntUnsignedSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+
+	if err := b.args[1].VecEvalInt(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	x := result.Uint64s()
+	y := buf.Uint64s()
+	result.MergeNulls(buf)
+	var res uint64
+	for i := 0; i < n; i++ {
+		if result.IsNull(i) {
+			continue
+		}
+		res = x[i] * y[i]
+		if x[i] != 0 && res/x[i] != y[i] {
+			result.SetNull(i, true)
```

Comment:
This line is redundant.
The vectorized framework need not to `SetNull` here.

Reply:
I have deleted that line
![123](https://user-images.githubusercontent.com/46105899/67255011-8e48c300-f4b2-11e9-9755-463358aafe0a.png)